RELEASING.md

```diff
@@ -10,20 +10,49 @@
 
 ## If there are no new Java dependencies
 
-6. Proceed to your working copy of systemsmanagement:Uyuni:Master and the package subscription matcher
+6. Branch `subscription-matcher` from `systemsmanagement:Uyuni:Master` checkout the package, and go to the checkout.
 7. Run `SMVER=X.YY; wget https://github.com/openSUSE/subscription-matcher/archive/v${SMVER}.tar.gz && tar -xzf v${SMVER}.tar.gz && mv subscription-matcher-${SMVER} subscription-matcher && tar -czf subscription-matcher.tar.gz subscription-matcher && rm -rf subscription-matcher v${SMVER}.tar.gz` (replace X.YY with the version you have at pom.xml)
 8. Change the version at subscription-matcher.spec
 9. Run `osc vc` and add changes
 10. Run `osc build` to check that the package is building
 11. Commit the changes with `osc ci`
-12. Copypac the package to all the Devel:Galaxy:Manager:X.Y projects with maintenance, with `iosc copypac openSUSE.org:systemsmanagement:Uyuni:Master subscription-matcher Devel:Galaxy:Manager:X.Y` or `osc -A https://api.suse.de copypac openSUSE.org:systemsmanagement:Uyuni:Master subscription-matcher Devel:Galaxy:Manager:X.Y`
+12. Verify that the package is still building in your branch
+13. Run `osc sr` to prepare a submit request
+14. Repeat for the 6-13 for the required `Devel:Galaxy:Manager:[Head|X.Y]` codestreams.
+15. Ping the release engineer.
+
+NOTE: `osc` only works by default for the public OBS instance (build.opensuse.org). Adjust your command to work with the internal one for `Devel:Galaxy:Manager:*` codestreams.
 
 ## If there are new Java dependencies
 
-6. Use [tetra](https://github.com/moio/tetra) to generate the main package and the kit package. Remember to adjust the SPEC file and changelog
-7. Copy the main package to your working copy of Devel:Galaxy:Manager:Head and the kit package to your working copy of Devel:Galaxy:Manager:Head:Kit
-8. Commit both packages with `iosc ci` if you have the alias or `osc -A https://api.suse.de` otherwise
-9. Copypac the main package to systemsmanagement:Uyuni:Master with `iosc copypac -t https://api.opensuse.org/ Devel:Galaxy:Manager:Head subscription-matcher systemsmanagement:Uyuni:Master` or `osc -A https://api.suse.de copypac -t https://api.opensuse.org/ Devel:Galaxy:Manager:Head subscription-matcher systemsmanagement:Uyuni:Master`
-10. Copypac the main package and the kit to all other Devel:Galaxy:Manager:X.Y projects with maintenance
+1. Use [tetra](https://github.com/moio/tetra) to generate the main package and the kit package. Remember to adjust the SPEC
+
+For Head and Uyuni:
+2. Branch `subscription-macher-kit` from `Devel:Galaxy:Manager:Head:Kit`, checkout and go to the checkout.
```

Comment:
Not sure why, but this is [not formatted as expected](https://github.com/uyuni-project/subscription-matcher/blob/19e20674dd5cb050aed122b4e43b5a4fcfc99268/RELEASING.md).